README.md

```diff
@@ -50,3 +50,30 @@ Django versions.
 
 You could also simply run regular ``tox`` in the root folder as well, but that would make testing the matrix of
 Python / Django versions a bit more tricky.
+
+# Work on the documentation
+
+Our documentation is generated by [Mkdocs](https://www.mkdocs.org).
+
+You can refer to their documentation on how to install it or use `mkdocs.sh` in
+this folder.
+
+It will run mkdocs in a [docker](https://www.docker.com/) container.
+
+Running the script without any params the `serve` command.
+The server is exposed on localhost on port 8000.
+If you want to configure port serve is exposed on you can use the
```

Comment:
Here I'm also not sure.. should it be "if you want to configure **a** port **where the** serve**r** is exposed on?